test/basic.html

```diff
@@ -131,6 +138,17 @@
         }, 5);
       });
 
+      test('toast is positioned according to its fitInto', function() {
+        var f = fixture('contained');
+        var container = f[0];
+        var toast = f[1];
+        toast.fitInto = container;
+        toast.center();
+        var rect = container.getBoundingClientRect();
+        assert.equal(toast.style.left, rect.left + 'px');
+        assert.equal(toast.style.bottom, (window.innerHeight - rect.bottom) + 'px');
```

Comment:
I'm not sure I understand this test -- your new code assigns `toast.style.left` to `fitInto.getBoundingClientRect().left`. Your test literally assings the `fitInfo`, and then basically checks that `toast.style.left` was assigned to `fitInto.getBoundingClientRect().left` after explicitly calling the code you wrote. I'd be very surprised if that wasn't the case, no matter what code you wrote :)

Instead, can we test that visually the toast is positioned where we expect it to? There's a fixed position on that div, is that where the toast should appear? Can we use those numbers instead?